chatimusmaximus/__init__.py

```diff
@@ -2,7 +2,48 @@
 import yaml
 import logging
 
-import plugins
+from websites import WebsitePlugin
+import simpleyapsy
+from simpleyapsy.file_getters import FilenameFileGetter
+from simpleyapsy.module_parsers import KeywordParser
+
+def validate_settings_not_blank(setting):
+    settings_have_values = False
+    for key, value in setting.items():
+        if value == str() or key == 'display_settings' or key == 'connect':
+            pass
+        else:
+            settings_have_values = True
+            break
+    return settings_have_values
+
+def instantiate_plugin_manager(settings):
+    file_dir = os.path.dirname(os.path.realpath(__file__))
+    website_path = os.path.join(file_dir, 'websites')
+    yapsy_file_locator = simpleyapsy.FileLocator(FilenameFileGetter())
```

Comment:
Instead of instantiating everything, let's use the setters methods to override the defaults.